source/features/copy-code-block.tsx

```diff
@@ -0,0 +1,31 @@
+import './copy-code-block.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import copyToClipboard from 'copy-text-to-clipboard';
+import features from '../libs/features';
+import {clippy} from '../libs/icons';
```

Comment:
The [convention](https://github.com/sindresorhus/refined-github/blob/2e4a24d3957c205c7bc49525c69e1fa08593d2ca/source/features/edit-readme.tsx#L4) is:

```suggestion
import * as icons from '../libs/icons';
```

And then use `{icons.clippy()}` later.